src/overlays/actors/ovl_En_Attack_Niw/z_en_attack_niw.c

```diff
@@ -202,14 +201,18 @@ void func_809B5670(EnAttackNiw* this, GlobalContext* globalCtx) {
     sp34.x = globalCtx->view.lookAt.x + tmpf1;
     sp34.y = globalCtx->view.lookAt.y + tmpf2;
     sp34.z = globalCtx->view.lookAt.z + tmpf3;
-
-    this->unk_2D4 = Math_Vec3f_Yaw(pos, &sp34);
-    this->unk_2D0 = (Math_Vec3f_Pitch(pos, &sp34) * -1.0f);
+    
+    if(1){}
```

Comment:
Make sure to run the formatter on all these files again, to fix issues like these

Reply:
should I just format everything while I'm at it?

Reply:
sure

Reply:
done